qa/workunits/suites/ffsb.sh

```diff
@@ -4,10 +4,8 @@ set -e
 
 mydir=`dirname $0`
 
-wget http://download.ceph.com/qa/ffsb.tar.bz2
-tar jxvf ffsb.tar.bz2
-cd ffsb-6.0-rc2
-patch -p1 < $mydir/ffsb.patch
+git clone https://github.com/ceph/ffsb.git
```

Comment:
This needs to be hosted on ceph.com. @djgalloway can you mirror this please?

Reply:
@lxbsz @batrick could create a release first to ease the mirroring.

Reply:
Or we can just use the `--depth 1` for the clone command instead ?

Reply:
https://github.com/ceph/ffsb/releases/tag/ceph_v1.0

How about that?

@lxbsz we try not to swarm github with our qa jobs so it's better to mirror a tarball.

Reply:
yeah, the .tar.gz would suffice.

Reply:
okay, lgtm.

Reply:
> This needs to be hosted on ceph.com. @djgalloway can you mirror this please?

Done

Reply:
@djgalloway 

BTW, what's the name of it and where ?

From `http://download.ceph.com/qa/` I only can see the following old ones:

```
ffsb-6.0-rc2.tar.bz2
ffsb.tar.bz2 
```

Not sure whether is still the firewall issue just likes the ffsb.git for me before, which has delayed half a day for me to see it.

Reply:
http://git.ceph.com/?p=ffsb.git;a=summary

Reply:
@batrick 
I suppose we will download the tarball, with this do you mean `git clone git://git.ceph.com/ffsb.git` ?

Reply:
Sure, I think you can try `--depth 1` too if that helps matters but there aren't many commits to begin with.

Reply:
Sure, will update this PR. Thanks

Reply:
Done. thanks.